Creating seed for existing models as per Trello task:

wfbraga opened the pull request:
Creating seed for existing models as per Trello task:
https://trello.com/c/LYtrGK43/7-criar-seed-populando-tabelas-existentes

wfbraga committed:
Creating seed for existing models as per Trello task:
https://trello.com/c/LYtrGK43/7-criar-seed-populando-tabelas-existentes

brunoao86 commented on db/seeds.rb:
```diff
@@ -5,3 +5,38 @@
+  RecurringBill.create(
+    description: Faker::Company.name,
+    amount: rand(100..300),
+    active: %i[active deactivated].sample,
```
```suggestion
    active: RecurringBill.actives.keys.sample,
```

brunoao86 commented on db/seeds.rb:
```diff
@@ -5,3 +5,38 @@
+    installment_amount: rand_total_amount / rand_installments_quantity,
+    current_installment: rand(1..rand_installments_quantity),
+    due_day: rand(1..20),
+    status: %i[active on_hold deactivated].sample,
```
```suggestion
    status: InstallmentPurchase.actives.keys.sample,
```

wfbraga committed:
creating relation between Groups, InstallmentPurchase and RecurringBill (#8)

* creating relation between Groups, InstallmentPurchase and RecurringBill as per Trello Task:
https://trello.com/c/wc7a41FH/8-criar-relacionamento-entre-group-e-installmentpurchase-recurringbill

Factory Creation for InstallmentPurchase
Test refactoring for InstallmentPurchase
Minor improvements on Group model and fatory
